optimize/backend/src/main/java/io/camunda/optimize/rest/security/ccsm/CCSMSecurityConfigurerAdapter.java

```diff
@@ -130,7 +130,8 @@ protected SecurityFilterChain configureWebSecurity(final HttpSecurity http) {
                       // public share related resources (API)
                       .requestMatchers(
                           new AntPathRequestMatcher(
-                              createApiPath(EXTERNAL_SUB_PATH + DEEP_SUB_PATH_ANY)))
+                              createApiPath(EXTERNAL_SUB_PATH + DEEP_SUB_PATH_ANY)),
+                          new AntPathRequestMatcher(EXTERNAL_SUB_PATH + "/api/**"))
```

Comment:
I had to add this to allow `/external/api/anything` URLs.

I don't know why it was working fine without this rule before. My guess is that the URL transformation from `/external/api/*` to `/api/external/*` was happening before getting here, with Jetty.

I am doing that transformation with a new filter now. See TomcatConfig.java.